.github/workflows/ci.yaml

```diff
@@ -1349,7 +1349,7 @@ jobs:
       - name: Run CI workflow generation checks
         run: |
           ./ci/actions-templates/gen-workflows.sh
-          git diff --quiet
+          git diff
```

Comment:
This is to make it clearer that `git diff` actually checks for coherence, and will actually print out the inconsistencies if there are any.